frontend/src/app/message.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+@Injectable()
+export class MessageService {
+    constructor(private http: HttpClient) { }
+
+    baseUrl = 'http://localhost:8000';
```

Comment:
can you make this an environment variable?

Reply:
done!